recipes/miniopy-async/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "miniopy-async" %}
+{% set version = "1.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/miniopy-async-{{ version }}.tar.gz
+  sha256: ffc1126a8bb48121a27ffa96cf03055f0dc7c4e0488841ea5eee61825d14a494
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - certifi
+    - aiofile
+    - aiohttp
+    - urllib3
+
+test:
+  imports:
+    - miniopy_async
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/hlf20010508/miniopy-async
+  summary: Asynchronous MinIO Client SDK for Python
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - L-ing
```

```suggestion
    - hlf20010508
```

@hlf20010508 This is supposed to be a GitHub username, so that the conda-forge bots can tag the maintainers for reviews on future releases.